conf/application.conf

```diff
@@ -312,11 +312,6 @@ microservice {
       context = "handle-subscription"
     }
 
-    pdf-generator {
```

Comment:
Does this need removed from respective app-config-env files where its overidden?

Reply:
Probably! Will do that once this PR is sorted